### PR TITLE
Added message_thread_id to url of chats with topics for message.get_u…

### LINKED ISSUE #1451: message.get_url() in chat with topics -> wrong url
### aiogram version

3.x

### Problem

The method message.get_url() returns wrong url when it call in chat with topics:
Returns `https://t.me/username/message_id`  but should `https://t.me/username/message_thread_id/message_id`

https://github.com/aiogram/aiogram/blob/acf52f468cae79b3511c2939cf39a801fa47f9f3/aiogram/types/message.py#L3750

I have not checked whether there is such a problem in other methods of the Message object. And I haven’t checked whether this method is used in other parts of the aiogram framework.

### Possible solution

possible patch 

```
--- a/.venv/lib/python3.10/site-packages/aiogram/types/message.py	
+++ b/.venv/lib/python3.10/site-packages/aiogram/types/message.py	(date 1712428103364)
@@ -3765,8 +3765,13 @@
             else self.chat.username
         )
 
+        message_value = (
+            f"{self.message_id}"
+            if not self.message_thread_id
+            else f"{self.message_thread_id}/{self.message_id}"
+        )
 
-        return f"https://t.me/{chat_value}/{self.message_id}"
+        return f"https://t.me/{chat_value}/{message_value}"
 
     def react(
         self,
```

### Alternatives

_No response_

### Code example

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Robotvasya**
# Description

Added message_thread_id to url of chats with topics for `message.get_url()` method.

Fixes #1451

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [x] pytest 
- [x] black
- [x] manual

**Test Configuration**:
* Operating System: Windows 
* Python version: Python 3.10

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Robotvasya** [comment]
Instead #1454  
I don't know how to merge to  discussion

**Comment by Olegt0rr** [comment]
@Robotvasya, do you need help writing tests or can you handle it yourself?

**Comment by Robotvasya** [comment]
I will try. And then ask for help)

**Comment by Robotvasya** [comment]
@Olegt0rr, I made some tests, please review it

File: tests/test_api/test_methods/test_get_url.py [hunk 1]
```diff
@@ -46,5 +46,61 @@ def test_method(
...
+            ["supergroup", -100123456789, None, True, True, None, "https://t.me/c/123456789/10"],
+            ["supergroup", -100123456789, None, True, True, 3, "https://t.me/c/123456789/3/10"],
+            ["supergroup", -100123456789, "name", False, False, None, "https://t.me/name/10"],
+            ["supergroup", -100123456789, "name", False, False, 3, "https://t.me/name/10"],
+            ["supergroup", -100123456789, "name", False, True, None, "https://t.me/name/10"],
+            ["supergroup", -100123456789, "name", False, True, 3, "https://t.me/name/3/10"],
+            ["supergroup", -10012345678, "name", True, False, None, "https://t.me/c/12345678/10"],
+            ["supergroup", -100123456789, "name", True, False, 3, "https://t.me/c/123456789/10"],
+            ["supergroup", -10012345678, "name", True, True, None, "https://t.me/c/12345678/10"],
+            ["supergroup", -100123456789, "name", True, True, 3, "https://t.me/c/123456789/3/10"],
```
Comment (Olegt0rr) [inline tests/test_api/test_methods/test_get_url.py:77]: For what purpose `chat_type` and `chat_id` are parametrized here?

Comment (Robotvasya) [reply tests/test_api/test_methods/test_get_url.py:77]: This creates different types of updates.
I planned that a “channel” type could be added here in the future. Although, as I understand it, the channels do not have topics.

File: tests/test_api/test_methods/test_get_url.py
Comment (Robotvasya) [inline tests/test_api/test_methods/test_get_url.py:1]: First group of tests covers all chat_types  cases.
Second covers if the supergroup chat is a forum.

**Review by JrooTJunior — APPROVED** [review]

**Pull request merged**